### DISCUSSION TITLE
Quarto Pub issue

### DISCUSSION THREAD

**Discussion by cjsrussell** [quarto-dev/quarto-cli#3627] [Q&A]
Hi All,

having an issue with Quarto Pub.

The publish command seems to work fine in the terminal, and there is even a correct thumbnail on the document on my Quarto Pub homepage, but when I click to view the document it is not available. The same if I try to go to the URL.

For context this is default new Quarto document that im using to test Quarto Pub, so everything is default.

![image](https://user-images.githubusercontent.com/42642025/206914371-42ac075a-1092-44d6-9eb1-37f33f726039.png)

![image](https://user-images.githubusercontent.com/42642025/206914482-f1dd06db-5d73-4e6f-8551-83ce32cde59f.png)

Thanks!

**Comment by cscheid** [comment]
@softwarenerd Do you have any input on this one?

**Comment by softwarenerd** [comment]
Hello,

https://cjsrussell.quarto.pub/

Is now working. 

There was a hiccup in the database related to your username. It was cjsrussell in one place and cjs_russell in another place. I'm not sure why this happened. I updated your account record, and now things are working.

Did you happen to receive any error messages when you were naming or renaming your username? I'm going to try to determine the root cause of the issue.

Best,

Brian

**Reply by cjsrussell** [reply]
Hey! 

Thanks for your help! I do not recall noticing any messages/warnings when setting my username, or at any other point (I am not sure if I manually changed my username), sorry.

Best,
Connair

**Reply by softwarenerd** [reply]
Hi Connair,

OK! This is the first time we've seen this issue come up. It seems as though there may have been a temporary hiccup writing to the database when you changed your username the last time.

Best,

Brian

**Comment by yello-data** [comment]
Hi,
Exactly the same problem happened to me.
I first had the username 'melloyello' but I changed to 'jmas'. I would like to keep 'jmas'.
Could you please fix the issue for me?
Thank you in advance.
Best,
